clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -754,6 +764,18 @@ llvm::Type *CodeGenTypes::ConvertType(QualType T) {
     llvm::Type *PointeeType = ConvertTypeForMem(ETy);
     if (PointeeType->isVoidTy())
       PointeeType = llvm::Type::getInt8Ty(getLLVMContext());
+    if (CGM.getTriple().isSPIRV() || CGM.getTriple().isSPIR()) {
+      const Type *ClangETy = ETy.getTypePtrOrNull();
+      if (ClangETy && ClangETy->isStructureOrClassType()) {
+      RecordDecl *RD = ClangETy->getAsCXXRecordDecl();
```

Comment:
Indentation in this block needs to be fixed.

Reply:
Thanks! Strange that clang-format didn't catch it

Reply:
@MrSidims, I don't know the reason for sure, but I guess it was effectively disabled by the changes @aelovikov-intel, recently made in GHA scripts in the `sycl` branch, which were missing in your branch.
I updated your branch and I see that lint check found more formatting issues.